src/v2/DevTools/renderUntil.tsx

```diff
@@ -1,5 +1,6 @@
 import { ReactWrapper, RenderUntilPredicate, mount } from "enzyme"
-import * as React from "react";
+import * as React from "react"
```

Comment:
is this `* as` needed anymore?

Reply:
This was a codemod addition when we added auto react imports. For some reason it included a bunch of these (and semis) in certain cases and so we've been getting diffs where it removes the semi.